ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-main.yaml

```diff
@@ -66,6 +59,140 @@ tests:
   skip_if_only_changed: ^\.tekton/|\.md$|^(LICENSE|OWNERS)$
   steps:
     workflow: openshift-ci-security
+- as: test-e2e-4-17
```

Comment:
If we should be testing the operator changes, shouldn't it be similar to what was done on release 1-0 for the operator repo?
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml#L81-L94